src/components/task-plan/homework.cjsx

```diff
@@ -227,6 +241,31 @@ ExerciseSummary = React.createClass
   removeTutorSelection: ->
     TaskPlanActions.updateTutorSelection(@props.planId, -1)
 
+  componentDidMount: () ->
+    el = @getDOMNode()
+    @staticPosition = @getPosition(el)
+    @handleScroll() # Update scroll position immediately on mount
+    @optimizedScrollFunc = throttle('scroll', 'optimizedScroll')
+    window.addEventListener('optimizedScroll', @handleScroll)
+
+  componentWillUnmount: () ->
+    window.removeEventListener('scroll', @optimizedScrollFunc)
+    window.removeEventListener('optimizedScroll', @handleScroll)
+
+  getPosition: (el) -> el.getBoundingClientRect().top - document.body.getBoundingClientRect().top
+
+  handleScroll: (e) ->
+    el = @getDOMNode()
+    if not el.textContent then return # HACK: Ignore empty span that is also an ExerciseSummary
+
+    if document.body.scrollTop + 60 > @staticPosition
+      el.classList.add('navbar', 'navbar-fixed-top', 'navbar-fixed-top-lower')
+      document.body.style.marginTop = '120px'
```

Comment:
Is there a reason this isn't a class on `<body>`? If so, can you include a description in the LESS file with a greppable string like `FANCY_SCROLL_MARGIN_TOP` in both places? I'm worried about having styling in the code that will end up out-of-sync or be difficult to find later on.